app/views/calendar/_save_location.html.erb

```diff
@@ -19,6 +20,7 @@
   data-undo-description="<%= undo_description %>"
   data-undo-button-aria-label="<%= undo_button_aria_label %>"
   data-undo-button-text="<%= undo_button_text %>"
+  <%= section_lang %>
```

Comment:
I'm not sure I understand the logic of this. `<%= section_lang %>` will be either `nil` when the locale is "en" or `"lang=en"` when the locale is not "en". Why do we want it to be `nil` when the locale is "en"?

Reply:
Good question @AlanGabbianelli

When a parent of the element already has `lang="en"` there's no added benefit in adding that to the child element – every child of that `lang="en"` container will be assumed to be in English unless it is specified otherwise.

On the other hand, since the element inherits `lang` from higher up the tree, when a parent of the element has `lang="cy"`, we need to add `lang="en"` to the element to ensure that screen readers will correctly read it in English. 

That is why I set that `section_lang` to nil when the locale is en –  in this instance, `section_lang` is only needed when the locale is not en.